src/clang-cpp-frontend/clang_cpp_language.cpp

```diff
@@ -44,9 +44,9 @@ void clang_cpp_languaget::force_file_type()
     !config.options.get_bool_option("no-library"))
   {
     log_debug("[CPP] Adding CPP includes: {}", ESBMC::esbmc_cpp_includes());
-    compiler_args.push_back("-isystem" + ESBMC::esbmc_cpp_includes());
+    compiler_args.push_back("-isystem=" + ESBMC::esbmc_cpp_includes());
     compiler_args.push_back("-I" + ESBMC::esbmc_cpp_includes() + "/CUDA");
-    compiler_args.push_back("-I" + ESBMC::esbmc_cpp_includes() + "/Qt/QtCore");
+    compiler_args.push_back("-I" + ESBMC::esbmc_cpp_includes() + "/Qt");
```

Comment:
I don't have the Qt version modelled here, but `pkg-config --cflags Qt5Core` gives me both paths:
```
-DQT_CORE_LIB -I/usr/include/qt5/QtCore -I/usr/include/qt5
```

Reply:
I added the qtcore back. I deleted some repeated OMs though (they didn't had ay specific behaviour)